modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/util/DLWebComponentProvider.java

```diff
@@ -78,8 +64,12 @@ protected void setIGDisplayContextProvider(
 
 	private static DLWebComponentProvider _dlWebComponentProvider;
 
+	@Reference
 	private DLAdminDisplayContextProvider _dlAdminDisplayContextProvider;
+
```

Comment:
@slnn You missed this reference. Also missed `IGDisplayContextProvider`'s setter method.

Reply:
Oh, sorry for my bad...